xlive/Blam/Engine/interface/screens/screen_squad_settings.cpp

```diff
@@ -631,12 +727,13 @@ void c_screen_squad_settings::update()
 
 	}
 
-	if (option_help_text_block)
+	if (option_help_text_block && value_string != _string_id_invalid)
```

Comment:
make the result of `value_string != _string_id_invalid` a bool that's set above and check that instead